src/components/EReceiptThumbnail.tsx

```diff
@@ -100,7 +106,8 @@ function EReceiptThumbnail({transaction}: EReceiptThumbnailProps) {
                         fill={secondaryColor}
                         additionalStyles={[styles.fullScreen]}
                     />
-                    {MCCIcon ? (
+                    {isThumbnail && fileExtension && <Text style={styles.labelStrong}>{fileExtension.toUpperCase()}</Text>}
```

Comment:
The label is not visible on the dark theme.

![Screenshot 2024-01-30 at 19 28 49](https://github.com/Expensify/App/assets/25520267/06f63658-c2b3-4807-b68e-9a035a3b2601)

![Screenshot 2024-01-30 at 19 23 11](https://github.com/Expensify/App/assets/25520267/a95ada81-a7b3-43c5-b62a-4eb4b1b7c0e1)